addon/services/stripev3.js

```diff
@@ -43,8 +43,13 @@ export default Service.extend({
       this.configure();
     }
   },
-
-  load(publishableKey = null) {
+ 
+  unload() {
+    this.set('didConfigure', false);
+    this.set('didLoad', false);
```

Comment:
In `configure`, the methods from the Stripe object are copied across to `this`. Totally agree with adding an `unload` method, in case you need to swap Stripe Connect accounts. However, I think unload should set all the methods that were copied across to `null` so that they are cleared out.